syntax = "proto3";

option java_package = "ex.grpc";

package dataproto;

service sensorService {
  rpc sendRequest (SensorDataMess) returns (SensorDataResponse) {}
  rpc loadData (EmptyMessage) returns (load){}
}

message SensorDataMess {
  string sensorMyData = 1;
  string sensorType = 2;
  string sensorTimestamp = 3;
  string id=4;
}

message SensorDataResponse{
  bool result = 1;
}

message EmptyMessage{
  string id=1;
}

message load{
  map<int32,SensorDataMess> sensorData = 1;
}


